README.md

```diff
@@ -78,8 +79,8 @@ $ npm install --save express ejs express-session body-parser cookie-parser
 The script [`./app.js`](app.js) creates, mounts, and exports an ExpressJS app instance.
 
 ```js
-const express = require('express');
-const bodyParser = require('body-parser');
+const express = require("express");
```

Comment:
We shouldn't update these to double quotes. This might be a default from your editor. Could you please revert this back to single quotes?